study-builder/studies/study-example.conf

```diff
@@ -4,7 +4,16 @@
     # The domain url, with a trailing slash at the end.
     "domain": "string",
     "mgmtClientId": "string",
-    "mgmtSecret": "string"
+    "mgmtSecret": "string",
+    # Optional. The password policy configured for the Auth0 database connection.
+    "passwordPolicy": {
```

Comment:
@charlottedye I don't think we need this for Prion since it's not on mobile, but just letting you know of this.